src/components/Card/index.js

```diff
@@ -16,7 +16,7 @@ const Card = ({
   to,
   ...props
 }) => {
-  const isLink = () => to !== null
+  const isLink = () => typeof to === 'string'
```

Comment:
What is this for?

Reply:
This possible needs to be looked at again, but is best dealt with in a separate issue. The prop demands that `to` be a `string` anyway, I just need to find the 'correct' way to check a prop value was passed in.

Reply:
Yeah - the PropTypes thing here: https://github.com/RaspberryPiFoundation/Bits/blob/bbe9e0e3098287500a099ea67121874696f06333/src/components/Card/index.js#L80 covers that - I don't *think* you need to do this extra check here.

Reply:
You can chain in an `isRequired` if you want to be extra sure on presence.